### PR TITLE
Search: Series big result

### PR DESCRIPTION
**Pull request by hypesystem**

### CODE REVIEW

**Comment by ITViking** [comment]
I really miss good ques/hints on testing this. On this instance url with a query that should pop up and show the big result would be nice

File: app/styles/series-page.scss [hunk 1]
```diff
@@ -133,7 +133,7 @@
 @for $i from 1 through 5 {
   .photo-stack__photo:nth-child(#{$i}) {
     scale: #{1 - 0.03 * $i};
-    transform: translateX(#{20 * $i - 40}px);
+    transform: translateX(#{5 * $i - 7.5}px);
```
Comment (ITViking) [inline app/styles/series-page.scss:136]: It's a little sad having three preview images lying on top of each other in such a manner that only the topmost is visible. 
Could we maybe calculate the size of them so the biggest is the bottom most picture?

**Comment by ITViking** [comment]
Good work man! Looks good :100: Had two comments :)

File: app/styles/series-page.scss (hunk 1 above)
Comment (hypesystem) [reply app/styles/series-page.scss:136]: Yeah I think something like that may be where we end up (cf. discussion on Trello with Signe), but this is *good enough* for now :smile:

**Comment by hypesystem** [comment]
> I really miss good ques/hints on testing this

Fair -- it's hard to give those, though, as it depends on what/how exactly you have indexed. A search for the exact title of one of your series should bring it up as the first result and highlight it :smile:

**Pull request merged**